src/controller.py

```diff
@@ -57,12 +74,23 @@ def _connect_signals(self):
         self.view.load_sett_action.triggered.connect(self.load_settings_file)
         self.view.slicing_info_action.triggered.connect(self.get_slicer_version)
 
-        self.view.calibration_action.triggered.connect(
-            self.calibrationPanel.show
-        )
-        self.view.bug_report.triggered.connect(
-            self.bugReportDialog.show
-        )
+        try:
+            self.view.calibration_action.triggered.connect(
+                self.calibrationPanel.show
+            )
+        except:
+            self.view.calibration_action.triggered.connect(
+                lambda: showInfoDialog(locales.getLocale().ErrorHardwareModule)
+            )
+
+        try:
+            self.view.bug_report.triggered.connect(
+                self.bugReportDialog.show
+            )
+        except:
+            self.view.bug_report.triggered.connect(
+                lambda: showInfoDialog(locales.getLocale().ErrorBugModule)
+            )
```

Comment:
I display information that submodules are not public when person without access tries to trigger them